src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpExchange.java

```diff
@@ -68,206 +74,227 @@
 
     /**
-     * Filter modules may store arbitrary objects with HttpExchange
-     * instances as an out-of-band communication mechanism. Other Filters
+     * {@Link Filter} modules may store arbitrary objects with {@code HttpExchange}
+     * instances as an out-of-band communication mechanism. Other filters
      * or the exchange handler may then access these objects.
-     * <p>
-     * Each Filter class will document the attributes which they make
+     *
+     * <p> Each {@code Filter} class will document the attributes which they make
      * available.
+     *
      * @param name the name of the attribute to retrieve
-     * @return the attribute object, or null if it does not exist
+     * @return the attribute object, or {@code null} if it does not exist
      * @throws NullPointerException if name is {@code null}
      */
-    public abstract Object getAttribute (String name) ;
+    public abstract Object getAttribute(String name);
 
     /**
-     * Filter modules may store arbitrary objects with HttpExchange
-     * instances as an out-of-band communication mechanism. Other Filters
+     * {@link Filter} modules may store arbitrary objects with {@code HttpExchange}
+     * instances as an out-of-band communication mechanism. Other filters
      * or the exchange handler may then access these objects.
-     * <p>
-     * Each Filter class will document the attributes which they make
+     *
+     * <p> Each {@code Filter} class will document the attributes which they make
      * available.
-     * @param name the name to associate with the attribute value
+     *
+     * @param name  the name to associate with the attribute value
      * @param value the object to store as the attribute value. {@code null}
-     * value is permitted.
+     *              value is permitted.
      * @throws NullPointerException if name is {@code null}
      */
-    public abstract void setAttribute (String name, Object value) ;
+    public abstract void setAttribute(String name, Object value);
 
     /**
-     * Used by Filters to wrap either (or both) of this exchange's InputStream
-     * and OutputStream, with the given filtered streams so
-     * that subsequent calls to {@link #getRequestBody()} will
-     * return the given {@link java.io.InputStream}, and calls to
-     * {@link #getResponseBody()} will return the given
-     * {@link java.io.OutputStream}. The streams provided to this
-     * call must wrap the original streams, and may be (but are not
-     * required to be) sub-classes of {@link java.io.FilterInputStream}
-     * and {@link java.io.FilterOutputStream}.
-     * @param i the filtered input stream to set as this object's inputstream,
-     *          or {@code null} if no change.
-     * @param o the filtered output stream to set as this object's outputstream,
-     *          or {@code null} if no change.
+     * Used by {@linkplain #Filter Filters} to wrap either (or both) of this exchange's InputStream
```

Comment:
Are you sure about the # here? Can you double check that the generated link is working? InputStream should be {@code InputStream}. OutputStream below should be {@code OutputStream}.

Reply:
Well spotted. I have fixed the link to the `Filter` class. I also changed the first appearance of `InputStream` and `OutputSteam` to  `{@link InputStream}` and `{@link OutputStream}` and subsequent occurrences to `{@code InputStream)` and `{@code OutputStream)`. The changes can be found in commit 3b4546e